### DISCUSSION TITLE
能否增加杜比全景声单独与视界单独开关

### DISCUSSION THREAD

**Discussion by ethanqiii** [Paladinfeng/MiaoProject#183] [Ideas]
亲爱的开发者你好，能否增加杜比全景声与视界分开的单独开关，这样更好区分up主上传的视频是否真的有杜比选项，现在为设置中仅有杜比视界开关，在观看视频时不知道是否已开启杜比视界，部分选成有全景声的视频也无法知晓是否已打开，添加的话会更加便捷查看当前播放视频的音视频格式，感谢

**Comment by Paladinfeng** [comment]
杜比全景声如果可以获取到，是会优先加载的，不需要单独的开关

**Reply by ethanqiii** [reply]
可问题就是…不知道是否获取到😂

**Reply by Paladinfeng** [reply]
不是那么重要了，有就自然会加载了，专心看视频就好了
